test/getMeals.test.js

```diff
@@ -0,0 +1,46 @@
+const supertest = require('supertest');
+const test = require('tape');
+const app = require('../server/app');
+
+module.exports = test('Testing for getMeals route', (t) => {
```

Comment:
remove the module.exports
```suggestion
test('Testing for getMeals route', (t) => {
```